symengine/cwrapper.h

```diff
@@ -246,6 +246,12 @@ void basic_subs(basic s, const basic e, const CMapBasicBasic * mapbb);
 //! in the given basic 'e' and returns it through basic 's'
 void basic_subs2(basic s, const basic e, const basic a, const basic b);
 
+//! Wrapper for ascii_art()
+
+//! Returns a new char pointer to the ascii_art string
+//! DO NOT forget to free it with 'basic_str_free'
```

Comment:
Don't use capital letters, it catches the eye too much. I would change the last sentence to:

`The caller is responsible to free the pointer using 'basic_str_free'.`